client/packages/system/src/Stock/DetailView/DetailView.tsx

```diff
@@ -99,27 +114,33 @@ export const StockLineDetailView: React.FC = () => {
     },
   ];
 
-  // useEffect(() => {
-  //   const listener: PluginEventListener = {
-  //     eventType: 'onChangeStockEditForm',
-  //     listener: onChange,
-  //   };
-
-  //   addEventListener(listener);
-
-  //   return () => removeEventListener(listener);
-  // }, [addEventListener, removeEventListener]);
-
   const footerProps = {
     isSaving: isUpdating,
     showSaveConfirmation,
     showCancelConfirmation,
-    isDirty,
+    disabled: isDirty && !hasPluginChanged,
```

Comment:
```suggestion
    disabled: !isDirty && !hasPluginChanged,
```
Only disable if both haven't changed?

Reply:
Oops -- yes 68ff74c06